Gopkg.toml

```diff
@@ -38,9 +38,9 @@ required = ["github.com/centrifuge/centrifuge-ethereum-contracts", "github.com/r
   name = "github.com/Masterminds/semver"
   version = "1.4.2"
 
-[[constraint]]
+[[override]]
```

Comment:
In protobufs we use precise-proof as well, is there any bad side effect of this that you could think about?

Reply:
No this should be fine.